src/_versions.ts

```diff
@@ -1,20 +1,18 @@
 export interface TsAppVersion {
-  version: string;
-  name: string;
-  description?: string;
-  versionLong?: string;
-  versionDate: string;
-  gitCommitHash?: string;
-  gitCommitDate?: string;
-  gitTag?: string;
-}
-
+    version: string;
+    name: string;
+    description?: string;
+    versionLong?: string;
+    versionDate: string;
+    gitCommitHash?: string;
+    gitCommitDate?: string;
+    gitTag?: string;
+};
 export const versions: TsAppVersion = {
-  version: "0.2.0",
-  name: "nachet-frontend",
-  versionDate: "2023-08-04T06:12:27.197Z",
-  gitCommitHash: "4c506f3",
-  versionLong: "0.1.0-4c506f3",
+    version: '0.1.0',
```

Comment:
original had 0.2.0, now it's 0.1.0. 

 0.1.0 < 0.2.0

this should never happen.

The source of the version is from package.json. does not appear to be updated.

Workflow should check that version is increasing always.

https://github.com/saitho/ts-appversion/issues/74

Reply:
The _versions.ts file gets overwritten on npm start everytime. If the version number in the package.json is 0.2.0 then this file will be automatically updated to version: '0.2.0' during runtime.

Reply:
yes, so it is required for the developer to update package.json version for each new version.

@ThomasCardin will soon enforce this with a github workflow check (making sure that the version of this PR is incremental)